### ISSUE TITLE
check-settings script: traceback when --global-docks False

### ISSUE THREAD

**Issue by maphew** [leo-editor/leo-editor#1515]
I get the attached traceback errors in Console when using _[check-settings]_ button in LeoSettings.leo. Selection from the messages:

~~~
createMenuFromConfigList NO PARENT Edit Settings doHandlersForTag,callTagHandler,onCreate,build_menu
hook failed: after-create-leo-frame, <function onCreate at 0x000001FE3F10EC80>, leo.plugins.viewrendered
....
AttributeError: 'NoneType' object has no attribute 'leo_docks'
~~~

Leo Log Window
Leo 6.2-b1-devel, devel branch, build fe91b27120
2020-02-28 08:24:39 -0600
Python 3.7.1, PyQt version 5.12.6
Windows 10 AMD64 (build 10.0.18362) SP0

**Comment by maphew** [comment]
Full console output: [leo-check-settings_iss1515.txt](https://github.com/leo-editor/leo-editor/files/4268998/leo-check-settings_iss1515.txt)

**Comment by maphew** [comment]
UX feedback: the _missing 111 xxx.xxx calls..._ messages would be better with colon instead of ellipsis, so: 
~~~
Missing 128. config.getBool calls:
    
    item-one
    item-two
    ...
~~~

(I spent some time figuring out if the _missing_ was talking about the lines above or below itself.)

**Comment by edreamleo** [comment]
Fixed the crash at 9a199ab in devel.
